src/pods/generate/TemplatedResourcesGenerator.ts

```diff
@@ -112,11 +137,18 @@ export class TemplatedResourcesGenerator implements ResourcesGenerator {
   Promise<Record<string, { link: TemplateResourceLink; meta?: TemplateResourceLink }>> {
     const files = await fsPromises.readdir(folderPath);
```

Comment:
Might be interesting to also print this array to see the order the files were processed?